test/unit/taxonomy/world_taxonomy_test.rb

```diff
@@ -7,14 +7,15 @@ def subject
     Taxonomy::WorldTaxonomy.new
   end
 
-  test "#world_taxons" do
+  test "#sort world_taxons alphabetically when children == 0, then alphabetically on world_taxons with children" do
```

Comment:
This doesn't quite make sense to me, perhaps just say `test "#order of world taxons"`?